<launch>

    <node pkg="arm_kinematics_tools" type="get_fk" name="get_fk_test" output="screen">
        <param name="kinematics_service_name" value="myrobot_manipulator_kinematics" />
    </node>

</launch>
